portfolio/src/main/java/com/google/sps/servlets/VideoTestingServlet.java

```diff
@@ -26,7 +26,7 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         try{
-            Video testVideo = AUX.VideoIdToObject(videoId);
+            Video testVideo = AUX.VideoIdToObject("asdfhkials");
             response.getWriter().println("Test video Title: " + testVideo.getTitle());
```

Comment:
Recommendation: Define a static final variable TEST_VIDEO_ID = "..."
and then reference it here.